git_repo/services/service.py

```diff
@@ -244,7 +244,10 @@ def format_path(self, repository, namespace=None, rw=False):
         if not rw and '/' in repo:
             return '{}/{}'.format(self.url_ro, repo)
         elif rw and '/' in repo:
-            return '{}:{}'.format(self.url_rw, repo)
+            if self.url_rw.startswith('ssh://'):
```

Comment:
could you update the commits related to that change the title of the commits to 'Adds missing ssh:// URL support' and in the body the details above? It's not only for GitBucket support, it's actually an important fix 😉